data/strings/strings.txt

```diff
@@ -15592,7 +15594,7 @@
 
   [error_enter_correct_instagram_page]
     tags = android
-    en = Enter a valid Instagram web address or account name
+    en = Enter a valid Instagram username or URL
```

Comment:
Instagram is a proprietary service that invades user privacy.  I suggest removing support.